flight/PiOS/Common/pios_bmp280.c

```diff
@@ -200,6 +195,36 @@ int32_t PIOS_BMP280_Init(const struct pios_bmp280_cfg *cfg, int32_t i2c_device)
 	return 0;
 }
 
+/**
+ * Initialise the BMP280 sensor
+ */
+#ifndef PIOS_EXCLUDE_BMP280_I2C
```

Comment:
Seems weird to use EXCLUDE for one and INCLUDE for the other. :/ I know it saves you diff but... ;)